### PR TITLE
Upgrade Gradle to 5.6.4

### PR DESCRIPTION
**Pull request by david-gang**
My intellij died  when working with gradle 4.10.
Soon gradle 6 will come out so this is quite ancient.
I tried to run all tasks except the publish task and they finished successfully.

### CODE REVIEW

**Comment by david-gang** [comment]
I upgraded shadow to 5.10 because i saw that it adds support to gradle5 in later versions

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Sat Nov 17 02:09:23 PST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```
Comment (ZacSweers) [inline gradle/wrapper/gradle-wrapper.properties]: Can you please keep this as all?

Comment (david-gang) [reply gradle/wrapper/gradle-wrapper.properties]: you mean the order of lines in the file ? Will leave it as is

Comment (david-gang) [reply gradle/wrapper/gradle-wrapper.properties]: ok

**Review by ZacSweers — APPROVED** [review]

**Comment by david-gang** [comment]
Please let me know if something is needed from my side. Squashing is possible from github now so i guess i won't need to rebase.

**Review by rharter — APPROVED** [review]
Looks good, thanks!

**Pull request merged**